services/softlayer_virtual_guest.go

```diff
@@ -68,6 +68,29 @@ func (slvgs *softLayer_Virtual_Guest_Service) CreateObject(template datatypes.So
 	return softLayer_Virtual_Guest, nil
 }
 
+func (slvgs *softLayer_Virtual_Guest_Service) ReloadOperatingSystem(instanceId int, template datatypes.Image_Template_Config) error {
+	parameter := [2]interface{}{"FORCE", template}
```

Comment:
There require to combine a request body like this:

{"parameters":["FORCE",{"imageTemplateId":"651103"}]}

FORCE means no need to manual confirmation